libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -693,7 +686,7 @@ uint32_t AP_Frsky_Telem::calc_batt(uint8_t instance)
  */
 uint32_t AP_Frsky_Telem::calc_ap_status(void)
 {
-    uint32_t ap_status;
+    uint32_t ap_status = 0;
```

Comment:
```suggestion
```

Reply:
Declare this the first time it is assigned to.  We try not to do pointless assignments where possible.